features/intl.yml

```
@@ -0,0 +1,107 @@
+name: Internationalization API
```

I've been waffling about how to handle the name here. The guidelines say to omit "API"

```suggestion
name: Internationalization
```

Though that's very generic. That said, it's a bit odd that the `Intl` group contains a bunch of features called `Intl.whatever` plus `Internationalization`. It might be better to name this `Intl`:

```suggestion
name: Intl
```

And optionally rename the _group_ as `Internationalization`.

I'll leave this up to you.